test/csuite/wt2323_join_visibility/main.c

```diff
@@ -210,10 +211,26 @@ test_join(TEST_OPTS *opts, SHARED_OPTS *sharedopts, bool bloom, bool sometimes_r
           insert_args[i].inserts, insert_args[i].removes, insert_args[i].notfounds,
           insert_args[i].rollbacks);
 
-    testutil_check(session->drop(session, sharedopts->posturi, NULL));
-    testutil_check(session->drop(session, sharedopts->baluri, NULL));
-    testutil_check(session->drop(session, sharedopts->flaguri, NULL));
-    testutil_check(session->drop(session, opts->uri, NULL));
+    while ((ret = session->drop(session, sharedopts->posturi, NULL)) == EBUSY)
```

Comment:
How about writing a `testutil_drop` that can be used everywhere instead of repeating the while logic?